.gitignore

```diff
@@ -2,3 +2,4 @@
 *.jl.cov
 *.jl.mem
 /Manifest.toml
+*.las
```

Comment:
We shouldn't ignore *.las files globally. Later we will have a `test/data` folder with las files that we can ignore. Please remove this entry form the `.gitignore` file.

Reply:
Okay!